### PR TITLE
zh-translation:/blog/2019/announcing-discuss.istio.io/index.md

### PR DESCRIPTION
**Pull request by ghl1024**
ref: [servicemesher/istio-official-translation/issues/1457](https://github.com/servicemesher/istio-official-translation/issues/1457)

### CODE REVIEW

File: content/zh/blog/2019/announcing-discuss.istio.io/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+publishdate: 2019-01-10
+aliases:
+    - /zh/news/2019/announcing-discuss.istio.io
+---
+
+在 Istio 论坛中的我们一直在努力寻找合适的方法，让用户与论坛的其他成员进行互动、交流，寻求其他用户的帮助，以及与该项目的开发者进行沟通。
+
+我们尝试了几种不同的途径，但是每种途径都有一些不足。 RocketChat 是我们最近的工作，但是缺少某些功能（例如，线程方面），意味着它不再适合围绕任何单一问题进行讨论。这也给一些用户带来了困惑：什么时候应该向 `istio-users@googlegroups.com` 发送电子邮件，什么时候又该使用 RocketChat ？
+
+我们认为我们已经在单一的平台上找到了正确平衡功能，因此我们很高兴地宣布 [discuss.istio.io](https://discuss.istio.io) 。这是一个功能全面的论坛，我们将从这里开始对 Istio 进行讨论沟通，它允许你提出问题并获得主题回复！真正的好处是，你可以使用你的 GitHub 身份进行这些操作。
```
Comment (tanjunchen) [inline content/zh/blog/2019/announcing-discuss.istio.io/index.md]: 因此我们很高兴地宣布 [discuss.istio.io](https://discuss.istio.io) 论坛成立啦

Comment (ghl1024) [reply content/zh/blog/2019/announcing-discuss.istio.io/index.md]: 已修改，@tanjunchen

**Comment by tanjunchen** [comment]
retest

**Pull request merged**